src/modules/temperature.cpp

```diff
@@ -4,14 +4,12 @@
 
 #if defined(__FreeBSD__)
 // clang-format off
```

Comment:
Can be dropped now as there's only one header, so no need to interfere with sorting by name.